package-builders/Dockerfile.opensuse_tumbleweed

```
@@ -0,0 +1,62 @@
+FROM opensuse/tumbleweed
```

```suggestion
FROM opensuse/tumbleweed:latest
```

Hadolint will still complain about that (because of using the `latest` tag, but it makes it clear that we explicitly want the most recent version of the image for this.